.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,48 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - develop
+      - main
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x, 15.x]
```

Comment:
These are all of the currently active LTS versions, and the most recent stable version.